chef-config/lib/chef-config/config.rb

```diff
@@ -690,6 +690,11 @@ def self.windows_home_path
       default :watchdog_timeout, 2 * (60 * 60) # 2 hours
     end
 
+    # Only use fips compliant algorithms
+    default( :fips_mode ) do
```

Comment:
Given that this is a constant, it's probably best to use `default :fips_mode, OpenSSL::OPENSSL_FIPS` (`do` blocks are for computed defaults).